.github/workflows/auto-open-pr.yml

```
@@ -0,0 +1,82 @@
+
+concurrency:
+  group: auto-pr-${{ github.ref }}
+  cancel-in-progress: false
+
+jobs:
+  open-pr:
+    name: Open PR to main if missing
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create PR via GitHub API (PAT)
+        if: ${{ secrets.GH_PR_TOKEN != '' }}
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)